src/transformers/models/falcon/modeling_falcon.py

```diff
@@ -415,7 +437,11 @@ def forward(
         else:
             present = None
 
-        attention_mask_float = (attention_mask * 1.0).masked_fill(attention_mask, float("-1e9")).to(query_layer.dtype)
```

Comment:
cc @Rocketknight1 this `1e-9` was causing problems in some numerical precisions (it would be converted to `-inf`)

Reply:
Ah, my bad!